runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsPlugin.java

```diff
@@ -183,7 +185,10 @@ public void onMenuOpened(final MenuOpened event)
 					.setTarget(entry.getTarget())
 					.setType(MenuAction.RUNELITE_SUBMENU);
 
-				for (Color color : invColors())
+				Set<Color> invEquipmentColors = new HashSet<>();
+				invEquipmentColors.addAll(invColors());
```

Comment:
It is okay to just `getTagsFromItemContainer(INVENTORY)` and `getTagsFromItemContainer(EQUIPMENT)` here